cmd/ipfs/main.go

```diff
@@ -453,10 +501,10 @@ func isClientError(err error) bool {
 
 	// cast to cmds.Error
 	switch e := err.(type) {
-	case *cmds.Error:
-		return e.Code == cmds.ErrClient
-	case cmds.Error:
-		return e.Code == cmds.ErrClient
+	case *cmdsutil.Error:
```

Comment:
do we actually get both pointer and non-pointer versions of this? I feel like we should try to normalize that down to just the pointer version